src/main/scala/com/fulcrumgenomics/basecalling/ExtractBasecallingParamsForPicard.scala

```diff
@@ -192,4 +192,18 @@ object BasecallingParams {
     val barcodeSeq = Seq(sample.i7IndexBases, sample.i5IndexBases).flatten.mkString
     output.resolve(PathUtil.sanitizeFileName(s"${sample.sampleName}.$barcodeSeq.$lane.bam"))
   }
+
+  /** Produces the path to the unmatched BAM file for the given lane.
+    *
+    * The BAM file name will be of the form "unmatched.<lane>.bam".  The resulting file name will be sanitized to
+    * remove problematic characters.
+    *
+    * @param output the base output directory.
+    * @param lane the lane number.
+    * @return the path to the unmatched BAM file for the given lane.
+    */
+  def unmatchedBamFileFrom(output: DirPath, lane: Int): PathToBam = {
```

Comment:
Looks good.  Could you plumb this in here: https://github.com/fulcrumgenomics/fgbio/blob/e1ac3b157c6abb2992a88d1a31f677329c8c05fc/src/main/scala/com/fulcrumgenomics/basecalling/ExtractBasecallingParamsForPicard.scala#L150

That way if we change this method, it will update the params file too?

Reply:
Bonus points for making the name `"unmatched"` a static variable in the object and using it instead of hard-code strings.